src/components/BooksForm.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
```

Comment:
Hi Guys, Good job so far.

Let's do some rearrangement.

Kindly move this file  into `src/container` dir

React bindings for Redux separate presentational components from container components. This approach can make your app easier to understand and allow you to more easily reuse components. In your components directory (presentational components) you have components that are not connected to the Redux store, and on your containers directory you have components that are connected to the Redux store. https://redux.js.org/basics/usage-with-react#presentational-and-container-components 

![image](https://user-images.githubusercontent.com/31897434/90045002-57080480-dcc6-11ea-8c39-129c865acf05.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/